### ISSUE TITLE
could not open requirements file error when specify requirements per function

### ISSUE THREAD

**Issue by lucasnp** [UnitedIncome/serverless-python-requirements#200]
I was trying to specify dependencies per functions in my project using serverless-python-requirements.

I ran into this error when packaging the functions:
"Could not open requirements file: [Errno 22] Invalid argument: ".serverless/function1\\requirements.txt"

After digging a little bit, it appears that the issue might with **lib\pip.js**. Function **dockerPathForWin** only replace first instance of backslash

```
function dockerPathForWin(options, path) {
  if (process.platform === 'win32' && options.dockerizePip) {
    return path.replace('\\', '/');
  }
  return path;
}
```

Should it be?

```
function dockerPathForWin(options, path) {
  if (process.platform === 'win32' && options.dockerizePip) {
    return path.replace(\//\g, '/');
  }
  return path;
}
```

**Comment by dschep** [comment]
Yes! Good catch. JS's `String.replace` is stupid :cry:

I've fixed this in master. Could you test it by installing the plugin with `npm i UnitedIncome/serverless-python-requirements` please?

**Comment by lucasnp** [comment]
Yep. It works now. Thanks!

**Comment by dschep** [comment]
excellent. released as 4.0.5!
